packages/babel-plugin-react-native-web/src/__tests__/__snapshots__/index-test.js.snap

```diff
@@ -7,11 +7,32 @@ export { ColorPropType, StyleSheet, Text, createElement } from 'react-native';
 
       ↓ ↓ ↓ ↓ ↓ ↓
 
-export { default as View } from 'react-native-web/dist/exports/View';
-export { default as ColorPropType } from 'react-native-web/dist/exports/ColorPropType';
-export { default as StyleSheet } from 'react-native-web/dist/exports/StyleSheet';
-export { default as Text } from 'react-native-web/dist/exports/Text';
-export { default as createElement } from 'react-native-web/dist/exports/createElement';
+\\"use strict\\";
+
+exports.__esModule = true;
+exports.createElement = exports.Text = exports.StyleSheet = exports.ColorPropType = exports.View = void 0;
```

Comment:
Ensure these snapshots are not changed. The commonjs flag is probably no longer needed here: https://github.com/necolas/react-native-web/blob/0.9.8/package.json#L18

Reply:
This was being caused by the `babel-core` bridge dependency